extensions/xcode/package.json

```diff
@@ -72,7 +75,18 @@
       "title": "Create Swift Playground",
       "subtitle": "Xcode",
       "description": "Create a new Swift Playground and open it directly in Xcode",
-      "mode": "view"
+      "mode": "view",
+      "preferences": [
+        {
+          "name": "playgroundDefaultLocation",
+          "type": "textfield",
+          "required": true,
```

Comment:
I guess `false` would be more suitable as the default value `~Desktop` should be okay for most of the users.

Reply:
In that case I choose to put `~/Desktop` as `default` on l85 which seems to fit more with the idea of turning this into a Raycast preference.

Reply:
I was thinking about disabling the `required` property to omit the Raycast preference configuration screen when opening the `Create Swift Playground` command for the first time

<img width="862" alt="Screenshot" src="https://user-images.githubusercontent.com/11733014/164744525-3563c0ce-2267-49b4-8b7e-eb6180256742.png">

As `~/Desktop` is a good default value for most of the users and therefore not truly required. 
Those users who want to specify a different a path can easily adjust the preference via the Raycast Extensions Tab.

Reply:
Thanks for the explanation on that point, I agree with you 😄